absl/copts/configure_copts.bzl

```diff
@@ -45,38 +44,11 @@ ABSL_DEFAULT_LINKOPTS = select({
 # ABSL_RANDOM_RANDEN_COPTS blaze copts flags which are required by each
 # environment to build an accelerated RandenHwAes library.
 ABSL_RANDOM_RANDEN_COPTS = select({
-    # APPLE
-    ":cpu_darwin_x86_64": ABSL_RANDOM_HWAES_X64_FLAGS,
-    ":cpu_darwin": ABSL_RANDOM_HWAES_X64_FLAGS,
-    ":cpu_x64_windows_msvc": ABSL_RANDOM_HWAES_MSVC_X64_FLAGS,
-    ":cpu_x64_windows": ABSL_RANDOM_HWAES_MSVC_X64_FLAGS,
-    ":cpu_k8": ABSL_RANDOM_HWAES_X64_FLAGS,
-    ":cpu_ppc": ["-mcrypto"],
-    ":cpu_aarch64": ABSL_RANDOM_HWAES_ARM64_FLAGS,
+    "@platforms//os:windows": ABSL_RANDOM_HWAES_MSVC_X64_FLAGS,
+    "@platforms//cpu:x86_64": ABSL_RANDOM_HWAES_X64_FLAGS,
```

Comment:
I think my summary of this pull request is "thanks for trying, I don't think this works, but it does let us know we have a problem."

The problem is that `@platforms//os:windows` and `@platforms//cpu:x86_64` can both be true. The options controlled by this are really about the compiler (not the OS), and the CPU.

I'm actually thinking that this logic should not be in this file at all. Since for this code we use runtime dispatch when we are not sure at compile time if the CPU supports certain instructions, I'm thinking that using on `[[target(foo)]]` in the source code might be a better solution.

cc: @laramiel

Reply:
> The problem is that @platforms//os:windows and @platforms//cpu:x86_64 can both be true.

Yea I was hoping CI could prove this out but I can't see the logs here. I've pushed a solution to that which uses `config_setting_group` + `match_all` to disambiguate.